source-mongodb/discovery.go

```diff
@@ -27,7 +27,7 @@ const (
 )
 
 type documentMetadata struct {
-	Op bool `json:"op,omitempty" jsonschema:"title=Change Operation,description=Change operation type: 'c' Create/Insert, 'u' Update, 'd' Delete."`
+	Op string `json:"op,omitempty" jsonschema:"title=Change Operation,description=Change operation type: 'c' Create/Insert, 'u' Update, 'd' Delete."`
```

Comment:
I don't think the description is rendering correctly in output collections; it gets cut off after the `,` so it just ends up being `"Change operation type: 'c' Create/Insert"`. Relatedly, it would be good to have a test that snapshots the discovered schema so we can more clearly it see it up front and prevent regressions.